nemo/collections/tts/modules/degli.py

```diff
@@ -0,0 +1,542 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Does this file contain code from https://github.com/Sytronik/deep-griffinlim-iteration?

Reply:
Not much, but yes (like the calc_loss function)

Reply:
Can you add their MIT license to this file? Like https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/modules/squeezewave.py

Reply:
I also have used code from the repository for implementing the dataset.

What should I do with the license in this case?
Add it to the entire file? or split the datalayers file to two (with different licenses)?

Reply:
Up to you both works.